docs/guides/tutorials/django.rst

```diff
@@ -0,0 +1,284 @@
+.. _ref_guide_django:
+
+======
+Django
+======
+
+Gel supports SQL protocol for communicating with the database. This makes it possible to use it with Django by matching a Django schema to the Gel schema. You don't even need to figure out the conversion yourself because we have an automated tool for that.
```

Comment:
```suggestion
|Gel| supports SQL protocol for communicating with the database. This makes it possible to use it with Django by matching a Django schema to the Gel schema. You don't even need to figure out the conversion yourself because we have an automated tool for that.
```

We have a new substitution thing that adds some popover hints for our new name. Anytime you have `Gel` written, it should be `|Gel|`.